.github/CONTRIBUTING.md

```diff
@@ -211,9 +211,10 @@ this is where our previously generated `client.pem` comes in:
 ```
 import httpx
 
-proxies = {"all": "http://127.0.0.1:8080/"}
+proxy = "http://127.0.0.1:8080"
```

Comment:
This example looks incorrect to me.
I'd assume the proxy address should be `https://`?